src/app/core/services/http-rundown.service.ts

```diff
@@ -15,27 +14,16 @@ export class HttpRundownService implements RundownService {
   constructor(
     private readonly http: HttpClient,
     private readonly httpErrorService: HttpErrorService,
-    private readonly entityParser: EntityParser,
-    private readonly responseParser: ResponseParser
+    private readonly entityParser: EntityParser
   ) {}
 
   public fetchRundown(rundownId: string): Observable<Rundown> {
-    return this.http.get<unknown>(`${RUNDOWN_URL}/${rundownId}`).pipe(
-      map(this.responseParser.parseResponse.bind(this.responseParser)),
-      map(this.assertType.bind(this)),
+    return this.http.get<{ data: unknown }>(`${RUNDOWN_URL}/${rundownId}`).pipe(
       catchError(error => this.httpErrorService.catchError(error)),
-      map(data => this.entityParser.parseRundown(data.rundown))
+      map(data => this.entityParser.parseRundown(data.data))
```

Comment:
`data.data` is a little (in lack of a better word) meh. 
What you are mapping from is technically the response, so you could name it `map(response => this.entityParser.parseRundown(response.data))`.

This goes for all the other places as well.

Reply:
Renamed as suggested.